app/Http/Resources/GalleryConfigs/FooterConfig.php

```diff
@@ -44,6 +46,11 @@ public function __construct()
 			$copy_right_year = $copy_right_year . '-' . $site_copyright_end;
 		}
 
-		$this->copyright = $copy_right_year !== '' ? sprintf(__('landing.copyright'), $site_owner, $copy_right_year) : '';
+		// Use custom copyright text if provided, otherwise use default format
+		if ($this->copyright_text !== '') {
```

Comment:
Can extract this if into a function so you can do `$this ->copyright= $this->getCopyright()` ?
That should allow for an early return and decrease the cyclomatic complexity. 🙂